js/views/modals/Settings/Settings.js

```diff
@@ -0,0 +1,87 @@
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import SettingsGeneral from './SettingsGeneral';
+import SettingsPage from './SettingsPage';
+import $ from 'jquery';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      modelContentClass: 'modalContent clrP border clrBr',
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+
+    this.currentTabView = '';
+    this.TabViews = { SettingsGeneral, SettingsPage };
+
+    if (this.options.removeOnClose) this.on('close', () => this.remove());
+  }
+
+  className() {
+    return `${super.className()} settings`;
+  }
+
+  events() {
+    return {
+      'click .js-tab': 'tabClick',
+      'click .js-cancel': 'cancelClick',
+      'click .js-save': 'saveClick',
+      ...super.events(),
+    };
+  }
+
+  tabClick(e) {
+    const targ = $(e.target).closest('.js-tab');
+    this.selectTab(targ);
+  }
+
+  selectTab(targ) {
+    const targTab = targ.data('tab');
+    const TabView = `Settings${targTab}`;
+    let newTabView;
+
+    this.$('.js-tab').removeClass('clrT active');
+    targ.addClass('clrT active');
+
+    if (this.TabViews[TabView]) {
+      newTabView = this[TabView] || new this.TabViews[TabView]();
```

Comment:
Can we pull out the `this[TabView]` check? Since we're never setting that anywhere, this will never be true. If we do decide to cache them at some point, then we could put it back in (although I think the instances should start with a lowercase letter. alternatively we could store them in an appropriately name variable, e.g. `this.tabViews` (lowercase t)).